presto-hive-hadoop2/bin/run_hive_wasb_with_azure_tests.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
always `#!/usr/bin/env bash` unless something is run in container, then `#!/bin/bash`